era5_land_forcing_data2008/README.md

```diff
@@ -0,0 +1,62 @@
+# ClimaLand forcing data - ERA5 reanalysis data for 2021
+This file holds one year of hourly data of a subset of ERA5 output fields.
+The fields are
+- Latitude ("lat") (from -90.0 to 90.0 degrees at a resolution of 1.0)
+- Longitude ("lon") (from 0.0 to 360.00 degrees at a resolution of 1.0)
+- Time ("time") (from 2008-01-01 to 2008-12-31)
+- 10 metre U wind component (lon, lat, time; "u10")
+- 10 metre V wind component (lon, lat, time; "v10")
+- Dew point temperature at 2 meters (lon, lat, time; "d2m")
+- Temperature at 2 meters (lon, lat, time; "t2m")
+- Surface air pressure (lon, lat, time; "sp")
+- Mean snowfall rate (lon, lat, time; "msr")
+- Mean surface direct short wave radiation flux (lon, lat, time; "msdrswrf")
+- Mean surface downward long wave radiation flux (lon, lat, time; "msdwlwrf")
+- Mean surface downward short wave radiation flux (lon, lat, time; "msdwswrf")
+- Mean total precipitation rate (lon, lat, time; "mtpr")
+- Leaf area index, high vegetation (lon, lat, time; "lai_hv")
+- Leaf area index, low vegetation (lon, lat, time; "lai_lv")
+
+## Prerequisites
+1. Python environment with cdsapi installed.
```

Comment:
Can you add a requirements.txt file or something that defines dependencies in a declarative way? You can see examples in some the land artifacts (@imreddyTeja can help)